src/dispatch/individual/service.py

```diff
@@ -74,6 +77,11 @@ def get_all(*, db_session) -> List[Optional[IndividualContact]]:
     return db_session.query(IndividualContact)
 
 
+@lru_cache(maxsize=1000)
+def fetch_individual_info(contact_plugin: PluginInstance, email: str, db_session: Session):
```

Comment:
What's the return type here, `dict`?

Reply:
Yeah, I think `dict[str, str]`. Although it depends on your plugin I suppose, so I didn't add it.